src/node/export.ts

```diff
@@ -2,6 +2,7 @@ const fs = require('fs');
 const fsPromises = require('fs/promises');
 const archiver = require('archiver');
 const path = require('path');
+const crypto = require('crypto');
```

Comment:
why was this added?

Reply:
so it generates random uuid, crypto is nodes module